src/applications/debt-letters/components/HowDoIPay.jsx

```diff
@@ -25,17 +23,11 @@ const HowDoIPay = () => (
     </p>
     <h3 className="vads-u-margin-top--1 vads-u-font-size--h5">By phone</h3>
     <p className="vads-u-margin-bottom--0 vads-u-margin-top--0">
-      Call us at{' '}
-      <a href="tel: 800-827-0648" aria-label="800. 8 2 7. 0648.">
-        800-827-0648
-      </a>
+      Call us at <Telephone contact="8008270648" />
       {'.'}
     </p>
     <p className="vads-u-margin-top--1">
-      If calling internationally, use{' '}
-      <a href="tel: 612-713-6415" aria-label="612. 7 1 3. 6415.">
-        612-713-6415
-      </a>
+      If calling internationally, use <Telephone contact="6127136415" />
```

Comment:
For international formats, please include this pattern: `pattern={PATTERNS.OUTSIDE_US}` - see https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/telephone/